contrib/CMakeLists.txt

```diff
@@ -22,4 +22,19 @@ if(BLAZE_COMPILER AND BLAZE_EVALUATOR)
     PRIVATE sourcemeta::blaze::compiler)
   target_link_libraries(sourcemeta_blaze_contrib_trace
     PRIVATE sourcemeta::blaze::evaluator)
+
+  if (UNIX)
```

Comment:
Wouldn't this fail on i.e. macOS? I think we should check for Linux specifically

Reply:
Good point. Will correct.